common/setups/serialization.py

```diff
@@ -71,17 +72,21 @@ class Import(SerializerObject):
 
     def __init__(
         self,
+        *,
         code_object_path: Union[str, FunctionType, Any],
+        unhashed_package_root: Optional[str] = None,
```

Comment:
I wonder if there maybe should be sth additionally like `extra_hash`, which the user can use to add explicitly to the hash, to make it unique and avoid possible collisions.
```suggestion
        unhashed_package_root: Optional[str] = None,
        extra_hash: Optional[str] = None,
```
This would be added in `_sis_hash` then.

Reply:
In principle, yes, this would have the same effect. But I'm not sure people are aware of that, and I thought that such extra arg here makes it more visible. But maybe instead we can also hint to this in the docstring of `unhashed_package_root`.

Reply:
Opinions from other people?

Reply:
I don't think it would be necessary, but I have nothing against adding an argument that I will never use.

Reply:
I put a comment